system/jlib/jstring.cpp

```diff
@@ -1058,7 +1070,7 @@ String::String(const char * value)
 String::String(const char * value, int offset, int _count)
 {
   text = (char *)malloc(_count+1);
-  memcpy(text, value+offset, _count);
+  memcpy_iflen(text, value+offset, _count);
```

Comment:
might be worth 4 space indenting this/other places in this class while your doing others.

Reply:
I think we should avoid making unrelated whitespace changes (or variable renames) in the same PR.  @richardkchapman can we discuss in the next triage meeting.

Reply:
I mentioned, because you had changed (corrected) the indentation on other lines in this class you had touched.
I tend to extend formatting fixes like this (or adding override) to the class I'm touching.

Reply:
yes.  I probably shouldn't have changed them either, but too tempting...